client/src/components/SignUp.js

```diff
@@ -1,41 +1,102 @@
-import React, { Component } from "react";
+import React, {Component} from "react";
+import {Alert} from 'reactstrap';
+import {post} from "../helper/Fetch";
+import {ReroutToLogin} from "./ReroutToLogin";
+
 export default class SignUp extends Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            email: "",
```

Comment:
The endpoint to make a new user in the database only needs a username field; it does not need an email also.